Documentation/markdown/IFC4.3XListOfChanges.md

```diff
@@ -0,0 +1,33 @@
+# List of changes between IFC4X3_RC4 and IFC4X3 versions
+
+### Missing data in OIP
+
+| Name                                 | IFC4X3_RC4.exp          | IFC4X3.exp            |
+|--------------------------------------|-------------------------|----------------------|
+| IfcCosine                            | `:heavy_check_mark:`    | `:x:`                |
+| IfcSine                              | `:heavy_check_mark:`    | `:x:`                |
+| IfcCosineSpiral                      |  `:x:`                  | `:heavy_check_mark:` |
+| IfcSineSpiral                        |  `:x:`                  | `:heavy_check_mark:` |
+
+
+### Entity definitions
+
+| Name                                         | IFC4X3_RC4.exp                                      | IFC4X3.exp                                                        |
+|----------------------------------------------|------------------------------------------------------------------------------------------------------------------------|
```

Comment:
you are missing a "|"
or this reason the table is not visualized correctly

Reply:
Resolved in [d292baf](https://github.com/tumcms/Open-Infra-Platform/pull/571/commits/d292baf520b65b2286c5b7a84cb3431f44a6a2f9)